infra.bs

```diff
@@ -1648,7 +1648,7 @@ need to be clear from context. This can be done by preceding the first instance
 to the <a>tuple</a>. An indexing syntax can be used by providing a zero-based index into a
 <a>tuple</a> inside square brackets. The index cannot be out-of-bounds.
 
-<div class=example id=example-tuple>
+<div class=example id=example-tuple-construction>
```

Comment:
Let's not change the ID here. Even if it would be slightly better looking at it now, it's generally preferable to keep IDs working so links people might have exchanged don't stop working.
```suggestion
<div class=example id=example-tuple>
```

Reply:
:+1: Yeah that makes sense